packages/flutter/lib/src/material/switch.dart

```diff
@@ -23,10 +39,25 @@ class Switch extends StatelessWidget {
     this.onChanged
   }) : super(key: key);
 
+  /// Whether this switch is on or off.
   final bool value;
+
+  /// The color to use when this switch is on.
+  ///
+  /// Defaults to accent color of the current theme.
   final Color activeColor;
+
+  /// A decoration to use for the thumb of this switch when the switch is on.
+  ///
+  /// Defaults to a circular piece of material.
   final Decoration activeThumbDecoration;
+
+  /// A decoration to use for the thumb of this switch when the switch is off.
+  ///
+  /// Defaults to a circular piece of material.
   final Decoration inactiveThumbDecoration;
+
+  /// Called when the user toggles with switch on or off.
```

Comment:
Maybe reiterate here that the caller will typically rebuild the widget in the opposite state.